zk-tests/src/Cheatcodes.t.sol

```diff
@@ -2,8 +2,10 @@
 pragma solidity ^0.8.13;
 
 import {Test, console2 as console} from "forge-std/Test.sol";
+import {MyToken} from "./ERC20.sol";
```

Comment:
I'd suggest using more of an inline dummy contract so it's obvious about the slots and not related to ERC20's context.

```solidity
contract FixedSlot {
    uint8 num; // slot index: 0
    
    function setNum(uint8 _num) public {
        num = _num;
    }
}
```